src/cli/cmds/build.ts

```diff
@@ -23,17 +23,22 @@ import Log from '../../lib/Log';
 import * as inquirer from 'inquirer';
 import {validatePassword} from '../inputHelpers';
 
-export async function build(config: Config, log = new Log('build')): Promise<void> {
-  const jdkHelper = new JdkHelper(process, config);
-  const androidSdkTools = new AndroidSdkTools(process, config, jdkHelper);
+/**
+ * Checks if the keystore password and the key password are part of the environment prompts the
+ * user for a password otherwise.
+ *
+ * @returns {Promise<[string, string]>} A promise with a tuple where the first item is they
+ * keystore password and the second is the key password.
+ */
+async function getPasswords(): Promise<[string, string]> {
```

Comment:
Could you create a class/interface for this with `keystorePassword` and `keyPassword` as members?

Reply:
done